src/device/radio/types.rs

```diff
@@ -6,6 +8,17 @@ pub enum Bandwidth {
     _500KHz,
 }
 
+/// Convert the bandwidth for use in the external lora-phy crate
+impl From<Bandwidth> for lora_phy::mod_params::Bandwidth {
+    fn from(bw: Bandwidth) -> Self {
+        match bw {
+            Bandwidth::_125KHz => lora_phy::mod_params::Bandwidth::_125KHz,
+            Bandwidth::_250KHz => lora_phy::mod_params::Bandwidth::_250KHz,
+            Bandwidth::_500KHz => lora_phy::mod_params::Bandwidth::_500KHz,
+        }
+    }
+}
+
```

Comment:
Either that or lorawan decides to use lora-phy exclusively as its LoRa physical layer,  probably leading to significant simplification, especially in removing the need for extra implementation files in lorawan-pilot to keep the option open to use other LoRa physical layer implementations.

It's too early to have that conversation, but I do not see any other LoRa physical layer implementation that is currently well supported or provides nearly the options for customization.  If a better LoRa physical layer implementation arises, it will need to support an API that is similar to that in lora-phy.  Converting to that better implementation should be straight-forward.